### DISCUSSION TITLE
Suggestion

### DISCUSSION THREAD

**Discussion by galacticjoel** [twbs/bootstrap#40510] [Ideas] [answered]
I have a suggestion regarding the Components section. It would be good if the HTML, CSS and JS code needed for a given component were provided for each component, so that you would not have to download the entire CSS and JS files.

**Comment by coliff** [comment, marked as answer]
They are already... if you're building the CSS from the SCSS files then you can choose which components to include or exclude. I do this with every Bootstrap site I work on. Likewise the JavaScript components are available separately in the source too.
